cmd/client/txgen/main.go

```diff
@@ -152,6 +170,9 @@ func main() {
 	clientNode.ServiceManagerSetup()
 	clientNode.RunServices()
 
+	log.Info("Harmony Client Node", "Role", clientNode.NodeConfig.Role(), "multiaddress", fmt.Sprintf("/ip4/%s/tcp/%s/p2p/%s", *ip, *port, nodeConfig.Host.GetID().Pretty()))
+	go clientNode.SupportSyncing()
```

Comment:
since I combined wallet and txgen as light client, if clientnode support syncing will make wallet work unexpected. we need a way to distinguish this two

Reply:
I actually don't know if this will work as expected. I'll remove it.